hokusai/production.yml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   strategy:
     rollingUpdate:
-      maxSurge: 1
+      maxSurge: 2
```

Comment:
i.e. start new pods in batches of 2 - this should reduce the time it takes for deploys / restarts to complete